demo/index.html

```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```

Comment:
Apologies. I was having a senior moment. Polymer no longer uses O.o() or the poyfill, but this component does (observe-js) :)